Cargo.toml

```diff
@@ -36,6 +36,7 @@ libtock_i2c_master = { path = "apis/peripherals/i2c_master" }
 libtock_i2c_master_slave = { path = "apis/peripherals/i2c_master_slave" }
 libtock_key_value = { path = "apis/storage/key_value" }
 libtock_leds = { path = "apis/interface/leds" }
+libtock_servo = { path = "apis/interface/servo" }
```

Comment:
Put this into alphabetical order (between `libtock_runtime` and `libtock_small_panic`).